src/main/java/edu/cornell/kfs/fp/businessobject/AmazonBillingCostCenterDTO.java

```diff
@@ -0,0 +1,97 @@
+package edu.cornell.kfs.fp.businessobject;
+
+import java.io.Serializable;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.sys.KFSConstants;
+
+public class AmazonBillingCostCenterDTO implements Serializable {
+
+    private static final long serialVersionUID = 2223891834883162575L;
+    
+    private String chartCode;
+    private String accountNumber;
+    private String subAccountNumber;
+    private String objectCode;
+    private String subObjectCode;
+    private String projectCode;
+    private String orgReferenceId;
+    
+    public AmazonBillingCostCenterDTO() {
+        String empty = "";
+        chartCode = empty;
+        accountNumber = empty;
+        subAccountNumber = empty;
+        objectCode = empty;
+        subObjectCode = empty;
+        projectCode = empty;
+        orgReferenceId = empty;
+    }
+    
+    public String getChartCode() {
+        return chartCode;
+    }
+    public void setChartCode(String chartCode) {
+        if (StringUtils.isNotBlank(chartCode)) {
+            this.chartCode = chartCode;
+        }
+    }
```

Comment:
looks like there are no lines between methods in this class, probably a formatter issue